src/main/resources/archetype-resources/silo/src/main/resources/application.yml

```diff
@@ -45,3 +45,4 @@ remoteTokenServices:
 consul:
     name: ${spring.config.name}
     agent: 192.168.33.10
+    siloPort: ${server.port}
```

Comment:
That is unfortunately imposed by Spring-boot, not from the developer: https://github.com/rebuy-de/green-checkout-silo/blob/master/silo/src/main/resources/application.yml#L13